autorest/templates/service_client.py.jinja2

```diff
@@ -20,7 +20,7 @@ from msrest import Serializer, Deserializer
 from {{ config_import }} import {{ code_model.client_name }}Configuration
 from azure.core.exceptions import map_error
 {%- for operation_group_name in operation_group_names %}
-{%- set camel_case_op_group_name = to_camel_case(get_client_name(operation_group_name)) %}
+{%- set camel_case_op_group_name = operation_group_name[1] %}
```

Comment:
I'm not super happy with how it looks like with `[0]`, let's get rid of "operation_group_names" completely and just do something like:
```
{%- for operation_group in operation_groups %}
...
:ivar {{ operation_group.python_case }}: {{ operation_group.camel_case__name }} operations
```

Reply:
switched thank, thanks!